src/components/People/People.tsx

```diff
@@ -0,0 +1,146 @@
+
+  useEffect(() => {
+    setLoading(true);
+
+    const fetchPoeple = async () => {
+      try {
+        const peopleFormServer = await getPeople();
+
+        setPeople(peopleFormServer);
+      } catch (e) {
+        setErrorMessage('Something went wrong');
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchPoeple();
+  }, []);
+
+  useEffect(() => {
+    const peopleWithParent = people.map(person => {
+      const mom = people.find(m => m.name === person.motherName);
+      const dad = people.find(d => d.name === person.fatherName);
+
+      return {
+        ...person,
+        mother: mom,
+        father: dad,
+        slug: getPersonLink(person.name, person.born),
+      };
+    });
+
+    setPeopleWithParents(peopleWithParent);
+  }, [people]);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {loading && <Loader />}
+
+          {errorMessage && !loading && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {errorMessage}
+            </p>
+          )}
+
+          {!loading && !errorMessage && people.length === 0 && (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          )}
+
+          {people.length !== 0 && (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {peopleWithParents.map(person => (
+                  <tr
+                    data-cy="person"
+                    className={classNames('', {
+                      'has-background-warning':
+                        normalizePersonName === person.slug,
+                    })}
+                    key={person.slug}
+                  >
+                    <td>
+                      <NavLink
+                        to={`../${getPersonLink(person.name, person.born)}`}
+                        className={getPersonClass(person)}
+                      >
+                        {person.name}
+                      </NavLink>
+                    </td>
+
+                    <td>{person.sex}</td>
+                    <td>{person.born}</td>
+                    <td>{person.died}</td>
+                    {person.mother ? (
+                      <td>
+                        <NavLink
+                          to={`../${getPersonLink(person.mother.name, person.mother.born)}`}
+                          className="has-text-danger"
+                        >
+                          {person.motherName}
```

Comment:
Ensure that `person.motherName` is correctly used. If `person.mother` is defined, `person.mother.name` should be used instead to ensure consistency.